### DISCUSSION TITLE
Doric Gen3 Minicube

### DISCUSSION THREAD

**Discussion by jrbagley** [pyPhotometry/code#16] [Q&A]
I was just wondering if Doric Gen3 cubes (w/ LED and drivers) would work with the pyphotometry board?

**Comment by ThomasAkam** [comment]
Hi @jrbagley,

The LED driver control inputs on the Doric Gen3 minicube cannot be connected to the LED driver outputs on the pyphotometry board, but see the discussion about connecting the LED driver inputs directly to the analog pins on the pyboard microcontroller here: https://github.com/orgs/pyPhotometry/discussions/14

**Reply by jrbagley** [reply]
Thank you!
